README.md

```diff
@@ -1,10 +1,165 @@
-# nuxeo-merritt
+# nuxeo_merritt
 
-Code for pushing Nuxeo content into Merritt.
+Code for pushing Nuxeo content into Merritt. The `nuxeo_merritt.py` script creates an ATOM feed for each collection that is set up in the Registry for deposit into Merritt. These ATOM feeds are stored on S3. 
 
-----------------
+The feed creation job is launched weekly by Airflow, which runs each feed creation task in parallel on ECS.
 
-##Interpreting the results of deposits from Nuxeo to Merritt
+## Basic Layout
+
+**nuxeo_merritt.py** - script that generates ATOM feeds for each collection configured in the [Registry](https://registry.cdlib.org/) for Merritt deposit. The script will only update the feed if there have been updates to the collection in Nuxeo since the last run.
+
+**run_nuxeo_merritt_task.py** - script for running `nuxeo_merritt.py` in Fargate (ECS).
+
+**Dockerfile** - Dockerfile used for building the image that's run in ECS (or locally, if you want)
+
+**sceptre/** - contains CloudFormation template for creating the AWS infrastructure needed to run the code in Fargate.
+
+**dags/** - contains the Airflow DAGs for the repo
+
+## Continuous Deployment
+
+When new code is pushed/merged into the main branch of this repo on github, 2 CodeBuild projects are triggered:
+
+1. `nuxeo_merritt-application-deploy` - builds the nuxeo merritt ATOM feed docker image and push to ECR
+2. `nuxeo_merritt-dags-deploy` - deploys DAGs and other code to Airflow
```

Comment:
What happens if one fails and one succeeds? Or if the dag deploy is faster than the docker image build?

Reply:
If the application deploy fails and the DAG deploy succeeds, then the DAG will launch an older version of the ECS task. It's possible that the ECS task will fail if there is some incompatibility between the DAG code and the application code. If the ECS deploy succeeds and the DAG deploy fails, then we might have the same sort of incompatibility issue until the DAG deploy is fixed. Either way, we can roll back to an earlier commit pretty easily, I think. This is this exact deploy model we're using for Rikolti btw, so it's potentially an issue there too...

If the DAG deploy is faster than the docker image build, then I don't think it's a problem unless we kick off the DAG right away without waiting to see that the task deploy finished. Right now the DAG is configured to run weekly on Saturday, so I doubt we'll run into this, but I'll add a note to the README.